news/31.bugfix

```diff
@@ -0,0 +1,4 @@
+Make it work with newwest Unidecode 1.0.22. Use its method instaed of duplicating it in here.
```

Comment:
Typo: `instaed` -> `instead`.
And I would not call it a bugfix, but a feature. But it is debatable.

Reply:
renamed and corrected